services/agora/.eslintrc.cjs

```diff
@@ -10,7 +12,7 @@ module.exports = {
   parserOptions: {
     parser: require.resolve("@typescript-eslint/parser"),
     extraFileExtensions: [".vue"],
-    // project: true,
+    project: ["tsconfig.json"],
```

Comment:
project: true is recommended online I think though I don't necessarily understand the intricacies of this option